### DISCUSSION TITLE
Irlanda e Nova Zelândia - Vistos e Validação

### DISCUSSION THREAD

**Discussion by ngustavo** [frontendbr/forum#542] [Discussão]
Olá, pessoal. Estou com dificuldades em achar informações na internet sobre a necessidade de validar diploma e para quais vistos isso é necessário. Essa parte é bem burocrática e difícil de entender nos mínimos detalhes.

A questão é que eu estou no 1º período de ADS (tecnólogo) e queria saber se consigo morar e trabalhar em um desses países. Sei que o tecnólogo não é aceito como lvl 7 na **NZ** e eu teria que aplicar para o _Work Visa_, mas a maioria dos empregadores pedem que você já tenha um visto. Já na **Irlanda**, eu vejo que uma oferta acima de _60k/y_ agiliza o processo.

**É melhor ter uma oferta antes de pedir o visto ou contrário?
Podem informar qual visto vocês usaram e qual vocês recomendam?**

Obrigado pela atenção :smile:

**Comment by laurindo** [comment]
Até onde eu sei, visto de trabalho(work visa) só é dado através da empresa
que está lhe contratando. Ela que deve dar entrada no processo. Uma outra
opção é tentar estudar pra conseguir trabalhar durante meio período.
Mas eu também não tenho muito conhecimento a respeito. Ainda estou lendo a
respeito.

Em algum tópico já foi falado sobre a questão de validar o certificado de
graduação. Tem umas dicas bem legais.

**Comment by felipefialho** [comment]
Linkando: #56 

E chamando @willianjusten
